### DISCUSSION TITLE
AdGuard Home encryption

### DISCUSSION THREAD

**Discussion by TroySigX** [AdguardTeam/AdGuardHome#4792] [Q&A]
Eventhough I have set encryption on AdGuard Home, it still doesnt listen on DoH. Can anyone help me with this?

**Comment by ainar-g** [comment]
How did you set up encryption and how do you check that it doesn't listen on the HTTPS port?  Can you provide a script that shows that?  E.g., using <https://github.com/ameshkov/dnslookup>.

**Reply by TroySigX** [reply]
I set up encryption by self-signing certificate for IP address, and set server name to local IP of the host device. Like this:![image](https://user-images.githubusercontent.com/88051085/182014240-ce139b02-eec0-4d80-8373-348b4b0405eb.png) ![image](https://user-images.githubusercontent.com/88051085/182014254-dea3535f-ace1-4f2e-b90c-9ab3afba88ee.png)

When I downloaded the DNS profile for IOS, it doesn't let me access the internet, I figured that was because IOS suppose DNS server doesn't exist.

**Reply by ainar-g** [reply]
The validation message says that the certificate is invalid.  If it is also invalid for the iOS device, the device won't use it.  Please provide the output of `dnslookup` queries.
